### PR TITLE
chore: update aws pod identity webhook

### PR DESCRIPTION
**Pull request by eliasscosta**
Fixes: https://github.com/kubernetes/kops/issues/16802

Update version of AWS EKS Pod Identity Webhook

### CODE REVIEW

File: upup/models/cloudup/resources/addons/eks-pod-identity-webhook.addons.k8s.io/k8s-1.16.yaml.template [hunk 1]
```diff
@@ -80,7 +80,7 @@ spec:
       serviceAccountName: pod-identity-webhook
       containers:
       - name: pod-identity-webhook
-        image: amazon/amazon-eks-pod-identity-webhook:v0.4.0
+        image: amazon/amazon-eks-pod-identity-webhook:v0.5.3
```
Comment (hakman) [inline upup/models/cloudup/resources/addons/eks-pod-identity-webhook.addons.k8s.io/k8s-1.16.yaml.template:83]: Why not `v0.5.7`?
https://github.com/aws/amazon-eks-pod-identity-webhook/releases/tag/v0.5.7

Comment (hakman) [reply upup/models/cloudup/resources/addons/eks-pod-identity-webhook.addons.k8s.io/k8s-1.16.yaml.template:83]: I would prefer the latest version in this case. Thanks!

**Comment by hakman** [comment]
@eliasscosta Looks good. Please squash the commits also. Thanks!

**Comment by eliasscosta** [comment]
> @eliasscosta Looks good. Please squash the commits also. Thanks!

done!

**Review by hakman — APPROVED** [review]
Thanks a lot @eliasscosta!

**Pull request merged**